modules/cnf-installing-the-n3000-operator.adoc

```diff
@@ -0,0 +1,162 @@
+    - vran-acceleration-operators
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-operatorgroup.yaml
+----
+
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest n3000 -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+stable
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `n3000-sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: n3000-subscription
+    namespace: vran-acceleration-operators
+spec:
+    channel: "<channel>"        <1>
+    name: n3000
+    source: certified-operators <2>
+    sourceNamespace: openshift-marketplace
+----
+<1> Specify the value for channel from the value obtained in the previous step for the `.status.defaultChannel` parameter.
+<2> You must specify the `certified-operators` value.
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-sub.yaml
+----
+
+. Verify that the OpenNESS Operator for Intel® FPGA PAC N3000 Operator is installed:
++
+[source,terminal]
+----
+$ oc get csv
+----
++
+.Example output
+[source,terminal]
+----
+NAME            DISPLAY                                         VERSION  REPLACES    PHASE
+n3000.v1.1.0    OpenNESS Operator for Intel® FPGA PAC N3000     1.1.0                Succeeded
+----
+
+You have now successfully installed the Operator.
+
+[id="install-n3000operator-web-console_{context}"]
+== Installing the OpenNESS Operator for Intel FPGA PAC N3000 Operator by using the web console
+
+As a cluster administrator, you can install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the web console.
+
+[NOTE]
+====
+You must create the `Namespace` and `OperatorGroup` CR as mentioned in the previous section.
+====
+
+.Procedure
+
+. Install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the {product-title} web console:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose *OpenNESS Operator for Intel® FPGA PAC N3000* from the list of available Operators, and then click *Install*.
+
+.. On the *Install Operator* page, select *All namespaces on the cluster*. Then, click *Install*.
+
+. Optional: Verify that the N3000 Operator is installed successfully:
+
+.. Switch to the *Operators* -> *Installed Operators* page.
+
+.. Ensure that *OpenNESS Operator for Intel® FPGA PAC N3000* is listed in the *vran-acceleration-operators* project with a *Status* of *InstallSucceeded*.
++
+[NOTE]
+====
+During installation, an Operator might display a *Failed* status. If the installation later succeeds with an *InstallSucceeded* message, you can ignore the *Failed* message.
+====
++
+If the Operator does not appear as installed, to troubleshoot further:
```

Comment:
Please use the same (and better) text that you have in the fec-operator file:

"If the console does not indicate that the Operator is installed, perform the following troubleshooting steps:"